Cake.BenchmarkDotNet/Dto/Percentiles.cs

```diff
@@ -1,19 +1,15 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace Cake.BenchmarkDotNet.Dto
+﻿namespace Cake.BenchmarkDotNet.Dto
 {
     public class Percentiles
     {
-        public double P0 { get; set; }
-        public double P25 { get; set; }
-        public double P50 { get; set; }
-        public double P67 { get; set; }
-        public double P80 { get; set; }
-        public double P85 { get; set; }
-        public double P90 { get; set; }
-        public double P95 { get; set; }
-        public double P100 { get; set; }
+        public long P0 { get; set; }

```

Comment:
Looking at the sample report you uploaded, these values should be double still 

ex: `341.33`

Reply:
Fixed